src/JSONManager.py

```diff
@@ -0,0 +1,68 @@
+import src.utils as utils
+import src.errors as errors
+import os
+
+
+class JSONManager(metaclass=utils.SingletonMeta):
```

Comment:
Usually in camel case abbreviations are used as words: so it would be JsonManager

Reply:
Actually just found that Python has a different convention: https://peps.python.org/pep-0008/ so we should keep JSONManager